apps/mobile/pushtracker/src/app/modules/profile-settings/profile-settings.component.ts

```diff
@@ -581,26 +584,24 @@ export class ProfileSettingsComponent implements OnInit {
             ProfileSettingsComponent.name,
             'Sending to pushtrackers: ' + pts.map(pt => pt.address)
           );
-          if (actionbar)
-            actionbar.updateWatchIcon({ data: PushTrackerState.unknown });
+          if (this.ptStatusButton)
+            this.ptStatusButton.state = PushTrackerState.unknown;
```

Comment:
We know that there are pts here, so we should go ahead and set this to be `busy`

```suggestion
            this.ptStatusButton.state = PushTrackerState.busy;
```